README.md

```
@@ -1,3 +1,82 @@
 # java-janggi
 
 장기 미션 저장소
+
+## 과제 진행 요구 사항
+
+### 게임판
+
+- 9행, 10열 크기의 게임판을 만든다.
+- 궁(宮) 또는 궁성(宮城)의 영역은 고려하지 않는다.
+    - 가로, 세로 이동만 고려한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)